TestAdapterforGoogle.TestDev17.yml

```diff
@@ -325,6 +325,18 @@ extends:
             Contents: Packaging.TAfGT.vsix
             TargetFolder: $(Build.ArtifactStagingDirectory)\drop
           continueOnError: true
+        # Pull a list only of files we build and ship to be scanned in FilesToScanDrop.
+        - task: PowerShell@2
+          displayName: 'Copy Scannable Files to: $(Build.ArtifactStagingDirectory)\FilesToScanDrop'
+          inputs:
+            filePath: './TestAdapterForGoogleTest/FilesToScan.ps1'
+            arguments: '-buildArtifactStagingDirectory $(Build.ArtifactStagingDirectory) -directoryToSearch $(Build.ArtifactStagingDirectory)\drop'
+        # Debug task for now to verify what files are in this FilesToScanDrop.
+        - task: 1ES.PublishPipelineArtifact@1
+          displayName: 'Publish Artifact: FilesToScanDrop'
+          inputs:
+            path: '$(Build.ArtifactStagingDirectory)\FilesToScanDrop'
+            artifact: FilesToScanDrop
```

Comment:
Once this is all confirmed to be working well, we should probably remove this so we're not uploading and storing any additional binaries when the pipeline runs.

Reply:
Will resolve in future small PR